server/templates/base_page.html

```diff
@@ -1,6 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <script async>
```

Comment:
@divad12 It looks like there may be a potential race condition here, if we don't set `window.env` before we try and load our config file for require.